nixos/tests/cagebreak.nix

```diff
@@ -17,56 +17,27 @@ in
   let
     alice = config.users.users.alice;
   in {
+    # Automatically login on tty1 as a normal user:
     imports = [ ./common/user-account.nix ];
+    services.getty.autologinUser = "alice";
+    programs.bash.loginShellInit = ''
+      if [ "$(tty)" = "/dev/tty1" ]; then
```

Comment:
Is this necessary at all in the test VM?

Reply:
No, I guess we should be able to drop it but I'd keep it to avoid potential issues in the future (e.g. when using `su -` which should also source that script).